packages/paste-website/src/pages/utilities/grid/index.mdx

```diff
@@ -0,0 +1,569 @@
+import {TwilioIcon} from '../../../components/icons/TwilioIcon';
+import {TWILIO_RED} from '../../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteUtility(filter: {name: {eq: "@twilio-paste/grid"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/grid/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Grid"
+  categoryRoute={SidebarCategoryRoutes.UTILITIES}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/utilities/grid"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/utilities-grid--grid-12-column-and-gutter-support"
+  abstractUrl="https://share.goabstract.com/3e7528f5-7981-421c-9308-e06d2f341c70?"
+  data={props.data.allPasteUtility.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+### About Grid
+
+Grid creates organized and consistent product layouts, by providing a standard for positioning elements. Grid speeds up the designer-to-developer workflow by improving the consistency of design implementations.
+Use a Grid to layout and align content by applying standard spacing between elements on a page or within a component.
+
+The standard desktop grid has 24-pixel gutters and can support up to 12 columns. The outside gutters are removed from the overall width of the grid.
+
+Grid is built using the [Flex component](/utilities/flex) and is responsive. Grid column widths grow and shrink based on the parent element width.
+
+## Examples
+
+### Equal column widths
+
+Use Column, which is a child component of Grid, to lay out equal-sized content, such as a feed of cards.
+
+<LivePreview scope={{Grid, Column, Card, Heading, Paragraph}} language="jsx">
+  {`<Grid gutter="space30">
+  <Column>
+    <Card padding="space70">
+      <Heading as="h4" variant="heading40">Instant Provisioning</Heading>
+      <Paragraph>The API and the Console allow you to search and provision local, toll-free, or mobile numbers by capability, geography, prefix or keyword.</Paragraph>
+    </Card>
+  </Column>
+  <Column>
+    <Card padding="space70">
+      <Heading as="h4" variant="heading40">Vast Inventory</Heading>
+      <Paragraph>Twilio maintains phone number inventory at a rate of 50 times the quantity provisioned daily for over 600 area codes and prefixes.</Paragraph>
+    </Card>
+  </Column>
+  <Column>
+    <Card padding="space70">
+      <Heading as="h4" variant="heading40">Quality Screening</Heading>
+      <Paragraph>Every phone number in the inventory passes a rigorous 60-day screening process to verify capabilities and test for unwanted call traffic.</Paragraph>
+    </Card>
+  </Column>
+</Grid>`}
+</LivePreview>
+
+### Unequal column widths, using column `span`
```

Comment:
Backticks can be removed here since it messes with the anchor on the heading.

Reply:
Good catch.